apps/web/src/pages/integrations/IntegrationsListModal.tsx

```diff
@@ -187,6 +187,7 @@ export function IntegrationsListModal({
         channel={provider?.channel}
       />
       <UpdateProviderSidebar
+        key={integrationIdToEdit}
```

Comment:
Fix the issue when the Update Integration Sidebar is not updated when the different integration is selected.